From ff444351661b9a4ce8025017d928c57ba638b0df Mon Sep 17 00:00:00 2001
From: Ahmad Abbas <ahmada@marvell.com>
Date: Thu, 14 Apr 2016 17:05:46 +0300
Subject: [PATCH 129/239] make: ble: updated the Makefile to work with yocto
 compiler

added stdlib includes as a parameter to the DDR Makefile.

Change-Id: Iad20614d6994fb18fbb16784bc94356808361fc6
Signed-off-by: Ahmad Abbas <ahmada@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29061
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 ble/ble.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ble/ble.mk b/ble/ble.mk
index 925f0e7..0eb2edf 100644
--- a/ble/ble.mk
+++ b/ble/ble.mk
@@ -40,9 +40,9 @@ BLE_SOURCES		+= 	ble/ble_main.c				\
 				drivers/delay_timer/delay_timer.c	\
 				$(PLAT_MARVELL)/common/plat_delay_timer.c
 
-PLAT_INCLUDES		+= 	-I$(MV_DDR_ROOT)
+PLAT_INCLUDES		+= 	-I$(MV_DDR_ROOT) -I$(CURDIR)/include/stdlib -I$(CURDIR)/include/stdlib/sys
 
 BLE_LINKERFILE		:=	ble/ble.ld.S
 
 $(MV_DDR_LIB):
-	@make -C $(MV_DDR_ROOT) --no-print-directory PLATFORM=$(PLAT) OBJ_DIR=$(CURDIR)/$(BUILD_PLAT)/ble
+	@make -C $(MV_DDR_ROOT) --no-print-directory PLAT_INCLUDES="$(PLAT_INCLUDES)" PLATFORM=$(PLAT) OBJ_DIR=$(CURDIR)/$(BUILD_PLAT)/ble
-- 
1.9.1

